pkg/doc/presexch/definition.go

```diff
@@ -298,8 +298,14 @@ func makeRequirement(requirements []*SubmissionRequirement, descriptors []*Input
 	return req, nil
 }
 
+// CredentialEntity contains the necessary information for processing the credential.
+type CredentialEntity struct {
```

Comment:
Can we rename this to "CredentialCandidate" to align a bit more with the text in the P-E spec? The term "candidate input" is used in section 5.3 but I think "CredentialCandidate" might be a better fit in this context.

Reply:
Sure.